src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_params.py

```diff
@@ -29,7 +28,9 @@
 policy_name_type = CLIArgumentType(help='Name of the backup policy.', options_list=['--policy-name', '-p'])
 job_name_type = CLIArgumentType(help='Name of the job.', options_list=['--name', '-n'])
 rp_name_type = CLIArgumentType(help='Name of the recovery point.', options_list=['--rp-name', '-r'])
-wl_type = CLIArgumentType(help = "Name of the workload type", arg_type=get_enum_type(['MSSQL', 'SAPHANA']), options_list=['--workload-type', '-wt'])
+backup_management_type = CLIArgumentType(help='Name of the backup management type.', arg_type=get_enum_type(['AzureWorkload', 'AzureIaasVM']), options_list=['--backup-management-type', '-bmt'])
```

Comment:
check with Kartik on shorthand parameter name for backup management type

Reply:
Not given in spec but added by me for easier debugging and will be modified before final commit alongwith with other help text specifications